.hooks/bootstrap

```diff
@@ -76,6 +76,12 @@ else
             echo "ERROR: virtualenv is missing. Please install using: pip install virtualenv" 1>&2
             exit 1
         fi
+        VIRTUALENV_VERSION=$(virtualenv --version)
+        if [[ $VIRTUALENV_VERSION != *16.7.* && "${SETUP_PY2}^^" != "NO" ]]; then
```

Comment:
why double `^`?

Reply:
I would just compare to "yes" as we do in the rest of the script:
```suggestion
        if [[ $VIRTUALENV_VERSION != *16.7.* && "${SETUP_PY2}" == "yes" ]]; then
```

Reply:
Also need to move this whole code block after the logic of `SETUP_PY2:-auto`. line 90

Reply:
^ - First letter upper case.
^^ - All letters upper case.
Typing proof.